data/korath/korath missions.txt

```diff
@@ -257,3 +270,207 @@ mission "Small Scale Delivery"
+			`	"We learned a lot from the Korath: even the most aggressive Unfettered are wary of attacking Hai planets, lest our species destroy itself like the Korath did. In all our raids in the Wah Ki system, never once have the Unfettered targeted civilian areas on Cloudfire." Batu appears shocked by this.`
+				goto questions
+			label somany
+			`	"The most advanced group of surviving Korath lives in systems near yours, according to pirate data. It seems the Korath have been raiding ships in pirate and Syndicate systems for food, while the pirates have been pillaging Korath ships for tech. I'm not sure why the Korath have limited themselves to piracy. They certainly have the firepower to dominate a few dozen of your planets.`
+			`	"From our race's records, it is quite unlike the Korath to be so close to a more primitive race without making demands. Their resources may be committed to another war, or they may be building up supplies for a shock and awe style attack. It's anyone's guess at this point.`
+			`	"Even the Unfettered are concerned about this. A few dozen human planets under the dominion of the Korath could make them a force to be reckoned with again. They could even be a threat to the Hai."`
+			choice
+				`	"Sounds foreboding."`
+					goto foreboding
+				`	"When do I get my cut of the spoils?"`
+			`	"Ha! I'd think you were a True Hai if it weren't for the missing fur. Don't worry, Captain, I'll make sure the others give you a suitable payment."`
+			`	Batu sighs as Choot'k continues. "We've made an unpleasant discovery recently. The Korath have stealth drones hiding in two pirate systems: Alcyone and Almach. They've been using these to spy on nearby traffic, including us. It's also... eh, Batu?"`
+				goto next
+			label foreboding
+			`	"Indeed it does, especially with our recent discovery. The Korath have stealth drones hiding in two pirate systems: Alcyone and Almach. They've been using these to spy on nearby traffic, including us. It's also... eh, Batu?"`
+			label next
+			`	Batu nods and pulls up something on his datapad. He reads mechanically to you. "The drone fleet acts as a phase array transceiver and hyperwave sounder. A phase array uses signals from multiple sources (the drones) transmitting or receiving slightly out of phase, to emulate a much larger antenna. It's the same principle as phase array radars used for meteorology and phase array telescopes for astronomy. Each drone's communication system is not much stronger than a human hyperspace relay, but the fleet acts together as one powerful relay. We think it's able to relay signals as far as 120 light-years, without the need for starlanes. What's most concerning is that relaying is only a secondary functionality; they're mostly observation systems.`
```

Comment:
The length of the text, the use of technical jargon, and Batu's confusion, emphasizes how advanced the tactic was that the Korath were using. I'm impressing upon readers the brilliance of the Korath's methods, and how much planning went into it. If the reader doesn't understand the jargon, then it'll feel like the Korath have done crazy technical magic, which is the intended effect. The reader does not need to understand the text, or even do more than skim over it, to achieve the desired effect.

On the off chance that a player knows what a phase array sounder and transceiver is, they'll still be impressed that the Korath made one that is nearly impossible to detect, out of drones that are constantly moving.

Reply:
By the way, what is going through Batu's head, is similar to what is going through yours. ("Why did Navy Intelligence have to explain this to me? I feel like they could have given me just the practical info. It would have been less of a text wall") That's why he is "reading mechanically," and he's only doing it because Choot'k wanted him to relay the information. Most likely, the main take-away Choot'k got from the long rambling explanation is what I pointed out: Korath made stealth drones using a brilliant technical feat that is beyond us.

Reply:
Yeah I suspected this was smth like that but just so you know I'd skip over the conversation at that jargon, expecting it to not really end
Still, I'm not expert at content creating ofc

Reply:
I shall await feedback from others on this subject. Does anyone have an opinion on whether I should make my point another way?

Reply:
```suggestion
			`	Batu nods and pulls up something on his datapad, which he begins to recite mechanically. "The drone fleet acts as a phase array transceiver and hyperwave sounder. A phase array uses signals from multiple sources (the drones) transmitting or receiving slightly out of phase, to emulate a much larger antenna. It's the same principle as phase array radars used for meteorology and phase array telescopes for astronomy. Each drone's communication system is not much stronger than a human hyperspace relay, but the fleet acts together as one powerful relay. We think it's able to relay signals as far as 120 light-years, without the need for starlanes. What's most concerning is that relaying is only a secondary functionality; they're mostly observation systems.`
```

That came through for me, but if we need to make it clearer...